providers/capv/standard/test_data/cluster_values.yaml

```diff
@@ -26,12 +26,13 @@ connectivity:
   bastion:
     enabled: true
   network:
+    allowAllEgress: true
```

Comment:
Just curious, why isn't this the default?

Reply:
I'd go with something that works and perhaps remove / set this to false once all the apps have the required policies (NetworkPolicy & CiliumNetworkPolicy) properly bundled. I am still hitting cases where it's not there yet. Today I hit that again with service for validating&mutating webhooks for CRD.

Reply:
Ah gotcha! It's for the Cilium cluster policy? I mistakenly thought it was some vsphere thing. Makes sense.